src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/Schemas/JSON/template.json

```diff
@@ -965,17 +965,15 @@
     "tags": {
       "description": "Common information about templates, these are effectively interchangeable with choice type parameter symbols",
       "type": "object",
+      "required": ["type"],
       "properties": {
         "language": {
           "description": "The programming language the template primarily contains or is intended for use with",
           "type": "string"
         },
         "type": {
-          "description": "The type of template. Commonly this is either project or item (and special checks are performed for these values to show only project or item type templates in certain contexts) but any string is allowed",
```

Comment:
@sayedihashimi @KathleenDollard just want to double check once again about removing option for value other than `item` or `project`:
- both help and https://docs.microsoft.com/dotnet/core/tools/dotnet-new mentions `other` types. If we are doing this change, we need to adjust documentation accordingly
- probably another alternative to keep definition as is to make description more precise:
`The type of template. The predefined values are: project, item. To use the template with Visual Studio only project or item type are allowed, otherwise template will not be shown in Visual Studio. CLI supports filtering based on project and item type only. For other uses, any string value can be used.`
At the moment, based on template search cache file, 252 templates have no type, 21 have type other than project or item.

Reply:
I think we should update the docs. I don't see any value in having other values here.